action.yaml

```diff
@@ -233,3 +245,16 @@ runs:
         OUTPUTS_FILE: ${{ steps.workspace.outputs.outputs-file }}
       run: cat "${OUTPUTS_FILE}" >> ${GITHUB_OUTPUT}
       shell: bash
+
+    - name: Fix permissions and ownership for files created with a container
+      if: ${{ inputs.container-image }}
```

Comment:
We might need some sort of `and always()` or something like that to ensure this always runs. Alternatively, you could use something like https://github.com/gacts/run-and-post-run to make this step a `post` task of the above step.